scenes/game/local_game.gd

```diff
@@ -18,6 +18,9 @@ const MinArenaLocation = 1
 const MaxArenaLocation = 9
 const ShuffleEnabled = true
 
+const log_player_color = "red"
```

Comment:
feels like a UI thing in game.gd rather than in the local_game logic
probably fine for now i guess, there's not a great split between UI/game logic for the log
but this is something I would expect to see defined in combat_log.gd or something

Reply:
probably a good idea; I can look into making it adjustable as well